### PR TITLE
#69 Problem bug solved and everything work

### PR DESCRIPTION
**Pull request by Mikivitkovski**

### CODE REVIEW

File: app/styles/globals/_colors.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 :root {
-  --bg-color: hsl(0, 0%, 11%);
+  --bg-color: #202023;
   --text-color: hsl(0, 0%, 100%);
```
Comment (edichoska) [inline app/styles/globals/_colors.scss:3]: why is this changed?

Comment (Mikivitkovski) [reply app/styles/globals/_colors.scss:3]: In figma is that code color for background and nav and footer i make my nav and footer in color what is in figma and i see difference in colors and i check again and i see that are not same colors in figma and in _colors.scss

File: components/reusable-components/navigation/navigation.module.scss [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+.navLink1 {
+  color: #ffff;
+  font-size: 20px;
+  margin-right: -400px;
+  margin-left: 450px;
+
+  /* Adjust the margin for space between links */
+}
+
```
Comment (edichoska) [inline components/reusable-components/navigation/navigation.module.scss:23]: remove all the comments in this file
